http.c

```diff
@@ -491,19 +490,7 @@ evhttp_maybe_add_date_header(struct evkeyvalq *headers)
 {
 	if (evhttp_find_header(headers, "Date") == NULL) {
 		char date[50];
```

Comment:
Yeah, and maybe it will be better to use `char date[sizeof("The longest possible date evert")];` instead of hardcoding it.